rel/i18n/emqx_mq_schema.hocon

```diff
@@ -113,6 +113,26 @@ emqx_mq_schema {
         label: "Key Expression"
     }
 
+    mq_individual_limits {
+        desc: """~
+            The byte size and count limits for an individual Message Queue."""
+        label: "Message Queue Limits"
+    }
+
+    max_shard_message_count {
+        desc: """~
+            The maximum number of messages in a shard for the Message Queue. The number of shards is configured
+            for all Message Queues in the Durable Storage settings."""
+        label: "Max Shard Message Count"
+    }
+
+    max_shard_message_bytes {
+        desc: """~
+            The maximum number of bytes in a shard for the Message Queue. The number of shards is configured
```

Comment:
should mention that the replication factor is not relevant to this limit? i.e. if a queue has 3 replicas, it does not mean the limit must be x3